modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java

```diff
@@ -205,12 +237,21 @@ public ListTypeEntry updateListTypeEntry(
 
 		listTypeEntry.setNameMap(nameMap);
 
-		ListTypeDefinition listTypeDefinition =
-			_listTypeDefinitionPersistence.findByPrimaryKey(
-				listTypeEntry.getListTypeDefinitionId());
+		if (!FeatureFlagManagerUtil.isEnabled(
+				listTypeEntry.getCompanyId(), "LPD-24055")) {
 
-		if (listTypeDefinition.isSystem() &&
-			!ObjectDefinitionUtil.isInvokerBundleAllowed()) {
+			ListTypeDefinition listTypeDefinition =
+				_listTypeDefinitionPersistence.findByPrimaryKey(
+					listTypeEntry.getListTypeDefinitionId());
+
+			if (listTypeDefinition.isSystem() &&
+				!ObjectDefinitionUtil.isInvokerBundleAllowed()) {
+
+				return listTypeEntryPersistence.update(listTypeEntry);
+			}
+		}
+		else if (listTypeEntry.isSystem() &&
+				 !ObjectDefinitionUtil.isInvokerBundleAllowed()) {
 
 			return listTypeEntryPersistence.update(listTypeEntry);
 		}
```

Comment:
@abesousa I'll approve this PR, but can you send a follow-up with a test to ensure the ERC won't be updated if the bundle is not allowed?

`ListTypeEntryLocalServiceTest` -> `testUpdateSystemListTypeEntry`